README.md

```diff
@@ -124,22 +124,26 @@ Repeat this procedure for every alert in the file:
 
 1. On the *Alert dashboard*, click `CREATE NEW ALERT` (more information [here](https://doc.linkurio.us/user-manual/latest/alert-dashboard/)).
 
-2. Copy the query from the file and paste it in the `Query` field
+2. Fill the `Alert name` field with the value provided in the file
 
-4. Click on `ADD A COLUMN` and fill the fields with the values provided in the file \
-*NOTE: repeat this step if more columns are specified in the file* \
+3. Fill the `Description` field with the description provided in the file
+
+4. Turn on, if it's not, the `Enable` toggle \
```

Comment:
I'll also add the step to share with all users otherwise the creator will be the only one able to work on it

Reply:
done